source/MultiGrid/MultiGrid.test.js

```diff
@@ -88,6 +113,19 @@ describe('MultiGrid', () => {
       })))
       expect(rendered.querySelectorAll('.ReactVirtualized__Grid').length).toEqual(2)
     })
+
+    it('should render 4 Grids when configured for fixed columns and rows', () => {
+      let multiGrid
+      findDOMNode(render(getMarkup({
+        fixedColumnCount: 1,
+        fixedRowCount: 1,
+        ref: (ref) => {
+          multiGrid = ref
+        }
+      })))
+
+      multiGrid.recomputeGridSize()
```

Comment:
When I first switched the measureAllCells calls to recomputeGridSize, negative values were getting passed as column and row indexes, which would throw an error. This was my (poor) failing test prior to adding the comparison against the fixed row/col sizes.